examples/default/Makefile

```diff
@@ -37,8 +37,8 @@ USEMODULE += saul_default
 BOARD_PROVIDES_NETIF := acd52832 airfy-beacon atmega256rfr2-xpro avr-rss2 b-l072z-lrwan1 cc2538dk fox \
         hamilton iotlab-m3 iotlab-a8-m3 lobaro-lorabox lsn50 mulle microbit msba2 \
         microduino-corerf native nrf51dk nrf51dongle nrf52dk nrf52840dk nrf52840-mdk nrf6310 \
-        nucleo-f767zi openmote-b openmote-cc2538 pba-d-01-kw2x remote-pa remote-reva \
-        ruuvitag samr21-xpro samr30-xpro spark-core telosb thingy52 yunjia-nrf51822 z1
+        nnucleo-f207z gucleo-f767zi openmote-b openmote-cc2538 pba-d-01-kw2x remote-pa \
```

Comment:
```suggestion
        nucleo-f207z nucleo-f767zi openmote-b openmote-cc2538 pba-d-01-kw2x remote-pa \
```

Reply:
Yes, I already noticed shortly after pushing that I made a typo there! Already fixed in the meantime ;)